docs/development/getting-started-locally.md

```diff
@@ -0,0 +1,137 @@
+# Etcd-Druid Local Setup
+
+This page aims to provide steps on how to setup Etcd-Druid locally with/without storage providers
+
+## Prerequisites 
+
+
+### [Optional] Installing [k9s](https://k9scli.io/) 
+
+[k9s](https://k9scli.io/) is an interactive terminal based UI to play with your kubernetes cluster. Makes it easy to interact with the cluster resources
+
+On Mac OS & Linux, install using brew
+
+```sh
+# via Homebrew
+brew install k9s
+```
+
+On Windows,
+```sh
+# via scoop
+scoop install k9s
+# via chocolatey
+choco install k9s
+```
+
+## Setting up Druid
+
+### Clone the Etcd-Druid Github repo
+    
+```sh
+# clone the repo
+git clone https://github.com/gardener/etcd-druid.git
+# cd into etcd-druid folder
+cd etcd-druid
+```
+
+---
+
+>Note: To setup Etcd-Druid with backups enabled on a [LocalStack](https://github.com/localstack/localstack) provider, refer [this](https://github.com/gardener/etcd-druid/blob/master/docs/development/getting-started-locally-localstack.md)
+
+---
+
+### Setting up the Kind Cluster
+
+```sh
+# Create a kind cluster
+make kind-up
+```
+
+This creates a new kind cluster and stores the kubeconfig in the  `./hack/e2e-test/infrastructure/kind/kubeconfig` file.
+
+To target this newly created cluster, set the `KUBECONFIG` environment variable to the kubeconfig file located at `./hack/e2e-test/infrastructure/kind/kubeconfig` by doing 
+```sh
+export KUBECONFIG=$PWD/hack/e2e-test/infrastructure/kind/kubeconfig
+```
+
+### Setting up Etcd-Druid
+
+* 
+    Generates the Etcd CRD and deploy etcd-druid into the cluster
+    ```sh
+    make deploy
+    ```
+
+### Without Backups enabled
+
+To setup Etcd-druid without backups enabled, apply the Etcd yaml to the cluster 
+```sh
+# Apply the druid yaml
+kubectl apply -f config/samples/druid_v1alpha1_etcd.yaml
+``` 
+
+Confirm that the `spec.backup.store` is commented out.
```

Comment:
This confirmation message should be moved to before the `kubectl apply...` command